engine/api/src/main/java/org/apache/cloudstack/engine/subsystem/api/storage/SnapshotInfo.java

```diff
@@ -18,10 +18,16 @@
 
 import java.util.List;
 
+import org.apache.cloudstack.framework.config.ConfigKey;
+
 import com.cloud.storage.Snapshot;
 import com.cloud.utils.exception.CloudRuntimeException;
 
 public interface SnapshotInfo extends DataObject, Snapshot {
+    ConfigKey<Boolean> BackupSnapshotAfterTakingSnapshot = new ConfigKey<Boolean>(Boolean.class, "snapshot.backup.to.secondary",  "Snapshots", "true", "Indicates whether to always"
```

Comment:
i think this should on a service - and not on a resource interface. I'd sugest `SnapshotManager`

Reply:
@DaanHoogland, it was in `SnapshotManager`. I had to move to `SnapshotInfo` to be able to use in `PresetVariableHelper`, which will be used in the `usage` too.